### PR TITLE
Update login template text

### LINKED ISSUE #20: Sign-in message problems
Hitting the prometheus route in OCP 3.7.0-0.143.0 results in a prompt:

"Sign in with a Account"

1) grammatically, it should be "an account"
2) Looking at tempates.go, it appears there should be a ProviderName insert in the message.   Maybe "Sign  in with an OpenShift account"?

### PR DESCRIPTION
**Pull request by mrogers950**
Fixes #20

### CODE REVIEW

File: templates.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func getTemplates() *template.Template {
 	{{ if .SignInMessage }}
 	<p>{{.SignInMessage}}</p>
 	{{ end}}
-	<button type="submit" class="btn">Sign in with a {{.ProviderName}} Account</button><br/>
+	<button type="submit" class="btn">Sign in with an OpenShift account</button><br/>
```
Comment (simo5) [inline templates.go]: should we substitute Openshift only if .ProviderName is not set ?

Comment (mrogers950) [reply templates.go]: Maybe, but would we ever have a different provider?

Comment (simo5) [reply templates.go]: It is possible people use another Oauth server in front of openshift for SSO, in that case they may want to specify to the user that those are the credentials being used/requested ?
Doesn't hurt to have the capability.

Comment (mrogers950) [reply templates.go]: That wouldn't change the provider name, oauth-proxy is only aware that it is using an "OpenShift" type provider by way of some URLs. Any more information about the OAuth server is doing is not exposed in any way that I know of..

Comment (simo5) [reply templates.go]: ah, ok, sad.

Comment (smarterclayton) [reply templates.go]: I think at this point we only want OpenShift.

Also, at some point we need to restyle this page (the UI team would do that for us and make it consistent with Jenkins oauth page).  Can you reach out to @spadgett and discuss when he might have time to do that (post 3.7 probably)?

**Pull request merged**